lib/configure.js

```diff
@@ -27,6 +32,28 @@ async function configure (gyp, argv) {
     // 'python' should be set by now
     process.env.PYTHON = python
 
+    const prefix = process.config.variables.node_prefix
```

Comment:
`prefix` is only used inside the following `if` conditional so could be moved there.

Reply:
@richardlau thanks, will move

Reply:
moved